cypress/integration/e2e/machine_spec.js

```diff
@@ -59,6 +59,41 @@ describe("State Machine Spec", () => {
             // Check Saving the design
             cy.wait("@patternSave");
         });
+        it("DO save design every time node attributes are configured",() => {
+            // Drop a node on canvas
+            cy.get(".component-drawer-svg-container[draggable='true']").first().click().drag("#cy-canvas-container", {force: true});
+            cy.wait(1000);
+            // Changing node attributes
+            cy.get("#Application_replicas").type(100);
+            // Check Saving the design
+            cy.wait("@patternSave");
+        });
+        it("DO save design with client-side validation failures",() => {
+            // Drop a node on canvas
+            cy.get(".component-drawer-svg-container[draggable='true']").first().click().drag("#cy-canvas-container", {force: true});
+            cy.wait(1000);
+            // Changing node attributes
+            cy.get("#Application_replicas").type(999999);
```

Comment:
is this id written somewhere?

Reply:
not exactly written but generated by the rjsf form